server/src/test/java/org/kaazing/gateway/server/config/parse/ServiceDefaultsTest.java

```diff
@@ -69,8 +69,8 @@ public void testDefaultConnectOptions() throws Exception {
         Assert.assertTrue("TLSv1".equals(sslProtocols[0]));
 
         Assert.assertTrue("en0".equals(connectOptionsMap.get("udp.interface")));
-        System.out.println(connectOptionsMap.get("tcp.transport"));
-        Assert.assertTrue("socks://localhost:8000".equals(connectOptionsMap.get("tcp.transport").toString().trim()));
+        final String trim = connectOptionsMap.get("tcp.transport").toString().trim();
```

Comment:
rename trim to tcpTransport, also remove final (I feel final for local variables is just noise).